source-map-support.js

```diff
@@ -378,8 +377,13 @@ function getErrorSource(error) {
     var contents = fileContentsCache[source];
 
     // Support files on disk
-    if (!contents && fs.existsSync(source)) {
-      contents = fs.readFileSync(source, 'utf8');
+    try {
+      const fs = require('fs');
```

Comment:
This should probably be `var` to not upset any old browsers? Maybe we can move the `try`/`catch` to the top of the file even 🤔 

```js
var fs
try {
  fs = require('fs')
} catch (_) {
  // Loading in catch block to convert webpack error to warning.
}
```
```js
    if (!contents && fs && fs.existsSync(source)) {
      contents = fs.readFileSync(source, 'utf8');
```